satpy/tests/reader_tests/test_seviri_l1b_nc.py

```diff
@@ -32,7 +32,7 @@
 from satpy.tests.utils import make_dataid
 
 
-def new_read_file(instance):
+def new_read_file(h5):
```

Comment:
`h5` isn't really an explicit name, could you rename this parameter?

Reply:
Will do.